src/aedes-instrumentation.ts

```diff
@@ -155,31 +167,44 @@ export class AedesInstrumentation extends InstrumentationBase {
 
   // #region span management
 
-  private addClientSpan(client: AedesClient, span: Span) {
+  // TODO: maybe handle case where client is undefined (meaning the span is linked to the broker) ?
```

Comment:
I would handle this too, broker has also an Id (more useful in cluster context), we could create a fake client with id `$BROKER_${broker.id}`

Reply:
ok, agree 👍🏼

Reply:
#5